src/org/labkey/test/HtmlFragmentSelection.java

```diff
@@ -0,0 +1,45 @@
+package org.labkey.test;
```

Comment:
Consider putting this in the `util` package
```suggestion
package org.labkey.test.util;
```